src/CoreWCF.Primitives/src/CoreWCF/OperationContext.cs

```diff
@@ -152,7 +152,7 @@ public MessageHeaders OutgoingMessageHeaders
             }
         }
 
-        internal bool HasOutgoingMessageProperties
+        public bool HasOutgoingMessageProperties
         {
```

Comment:
Leave as internal. Replace the usage with this:
```c#
OperationContext.Current.OutgoingMessageProperties.Count > 0;
```

Reply:
Addressed in cccab6cab06f04ace4bc7903cc87da2a48c28c5f